sdk/src/poh_config.rs

```diff
@@ -15,10 +15,30 @@ pub struct PohConfig {
     /// None enables "Low power mode", which implies:
     /// * sleep for `target_tick_duration` instead of hashing
     /// * the number of hashes per tick will be variable
-    pub hashes_per_tick: Option<u64>,
+    hashes_per_tick: Option<u64>,
```

Comment:
let's revert this member privacy, the getter/setter/new and all related changes.  it breaks the `*Config` ergonomics that are expected elsewhere in the code base

Reply:
i was thinking we just reinitialize the service with a new `PohConfig`